### DISCUSSION TITLE
LQAS duplicate check to be updated

### DISCUSSION THREAD

**Discussion by naveeddil** [xlg8/APMIS-Project#662] [General]
As a rule, we can assign a surveyor to collect multiple samplles in a single Cluster due to population density. So we need to put a validation on the LQAS data. The query should check **Lot Number & Lot cluster Number** (blue circled) jointly for uniqueness although the current condition is on **Lot Number & Cluster**. (red circle)

The error comes when you try to change/ correct the lot number as mentioned in the screenshot below. The lot number should be changed to 7 instead of 16 (data entry error by user). When I try to correct it, the error appears which is correct but the query/conditions behind the query are incorrect.
![LQAS Error](https://github.com/AFG-Polio-Data/APMIS-Project/assets/54671296/6ab1f7dc-eb2d-4170-a744-df0649cf25a4)

You can also seee the data view where the Lot cluster numbers are unique.
![data](https://github.com/AFG-Polio-Data/APMIS-Project/assets/54671296/cd017677-6601-40ee-afc6-836fd151ddb0)

**Comment by naveeddil** [comment]
@danieltomabba and @RaziaMahmodi  Looking forward!

**Reply by RaziaMahmodi** [reply]
Thanks @naveeddil working on this issue is ongoing. I keep you posted with further updates.
